### DISCUSSION TITLE
Π2020159 - 6ο Παραδοτέο

### DISCUSSION THREAD

**Discussion by AlexIoanP2020159** [courses-ionio/sw#1505] [Show and tell]
[Αναφορά](https://github.com/AlexIoanP2020159/sw/tree/2020159/projects/2020159)

**Comment by AndreasKarageorgiou** [comment]
Καλησπέρα συνάδελφε , νομίζω πως θα ήταν καλύτερο για πλήρη βαθμολογηση στο show and tell να βάζεις ολόκληρο το παραδοτέο και όχι μονο το λινκ της αναφοράς.
